ACE/ace/Condition_Recursive_Thread_Mutex.h

```diff
@@ -23,12 +23,16 @@
 #if !defined (ACE_HAS_THREADS)
 #  include "ace/Null_Condition.h"
 #else /* ACE_HAS_THREADS */
+// ACE platform supports some form of threading.
+
 #include "ace/Recursive_Thread_Mutex.h"
 #include "ace/Condition_Attributes.h"
 #include "ace/Condition_T.h"
 
 ACE_BEGIN_VERSIONED_NAMESPACE_DECL
 
+class ACE_Time_Value;
```

Comment:
this is a 'forward declaration'; the header does not #include ace/Time_Value.h, even though it is referencing ACE_Time_Value*. Depending on what the other #includes are pulling in, this may not be needed (it's just more correct this way).